students/Lincoln_Zhang/Session9/cli_main.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+import sys,os,textwrap,csv
+from donor_models import donor_db_collection
+from donor_models import donor
+
+
+
+def donation_accept():
+    donor_name = input("Please tell me your name: ")
+    donor_name = donor_name.capitalize()
+
+    try:
+        donation_amount =list(int(input("Please tell me how much you would like to donate: ")))
```

Comment:
you may want to try [(int(input("Please tell me how much you would like to donate: "))] instead, otherwise this throws an error